lib/ios-deploy.js

```diff
@@ -72,6 +72,7 @@ class IOSDeploy {
         } else {
           const readStream = fs.createReadStream(itemPath, {autoClose: true});
           const writeStream = await afcService.createWriteStream(pathOnPhone, {autoDestroy: true });
+          writeStream.on('finish', writeStream.destroy);
```

Comment:
which argument `writeStream.destroy` is supposed to receive?

Reply:
Nothing since it ended properly